docs/configuration.md

```diff
@@ -227,6 +225,22 @@ Options:
   -h, --help
           Print help
 
+Miscellaneous:
```

Comment:
What's the motivation for moving Miscellaneous above rule and file selection?

Reply:
Mistake.